tree-sitter.json

```diff
@@ -13,10 +13,16 @@
   ],
   "metadata": {
     "version": "0.0.1",
-    "license": "ISC",
-    "description": "",
+    "license": "MIT",
+    "description": "Crystal grammar for tree-sitter",
+    "authors": [
+      {
+        "name": "Gabriel Holodak",
+        "email": "gthepiper@gmail.com"
+      }
+    ],
```

Comment:
The authors array could also be updated :slightly_smiling_face:

Reply:
I agree, just don't know who should go in there, as there were a handful of people who contributed to your original repo who aren't as active anymore.

Should we just go with us 4 for now? (Hugo, Devnote, me, and you) That's who have contributor-level access to the repo currently.

Reply:
That works for me! One of those old contributors just made a single PR, and the other doesn't actually exist (I  commited with a misconfigured email setting :roll_eyes: )